packages/user_packages/README.md

```diff
@@ -15,7 +15,8 @@ IoTCC over MQTT Protocol to acquire "registered edge system", i.e. iotcc_edge_sy
 which is pre-loaded during the installation in order to provide the required credentials and configuration parameters in the liota package manager registry.
 The properties for the edge system can be set as 'key:value' pair, you can also set the location by passing the
 'latitude:value' and 'longitude:value' as properties in the user package.
-Kindly include the unregister edge_system call in the clean up method required during the unload of the package.
+Kindly include the unregister edge_system call in the clean up method required during the unload of the package. The retry mechanism has been implemented for important
```

Comment:
Instead of saying "Kindly include the unregister call", for better understanding, we should say, "if unregister call is included in the cleanup method, the registered object can be unregistered and hence deleted from the server side."
 
I also noticed that you have provided a good comment in the package, so i think Readme should also reflect that. (You can reuse that)

Reply:
Added